packages/cli/src/handlers/dbt/compile.ts

```diff
@@ -80,8 +80,13 @@ export const dbtList = async (
             'unique_id',
         ];
         const version = await getDbtVersion();
-        // older dbt versions don't support --quiet flag
-        if (!version.startsWith('1.3.') && !version.startsWith('1.4.')) {
+        // older dbt versions don't support --quiet flag - only use it if the version is 1.6.6 or higher
+        if (
+            version.localeCompare('1.6.6', undefined, {
```

Comment:
I'd prefer to use a specific `semver` package for this comparison, or at least extract `localeCompare` into a `compareSemver` function that has some unit tests against it.

Reply:
sure! On it